doc/articles/features/ElevatedView.md

```diff
@@ -0,0 +1,49 @@
+# ElevatedView
+On many design languages like _Material Design_, there's a notion of elevation where a portion of the UI should be presented has been _elevated_ over the rest of the content.
+
+In this case, the UWP proposed solution can't work on all platforms because of technical limitations. To address this problem, Uno provides a control called `ElevatedView`, able to produce a similar result on all platforms, providing an elevated result.
+
+This control is very useful to create cards with both rounded corners and elevated effect - which could be challenging to produce on some platforms.
```

Comment:
```suggestion
This control is very useful to create cards with both rounded corners and an elevated effect - which could otherwise be challenging to produce on some platforms.
```